opensearch/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG TAG=2.11.0
+FROM opensearchstaging/opensearch:$TAG
```

Comment:
I don't think you want to use 'opensearchstaging'.  I think this is only used for release candidates.  The correct namespace is 'opensearchproject'.

Reply:
Done.